components/centraldashboard/public/components/main-page.css

```diff
@@ -78,60 +82,38 @@ app-drawer .footer>.build {
     color: rgba(255, 255, 255, 0.64)
 }
 
-#NamespaceSelector {
-    position: absolute;
-    right: 1em;
-    top: 50%;
-    transform: translateY(-50%)
-}
-
 app-toolbar {
-    display: flex;
-    justify-content: center;
+    @apply --layout-horizontal;
+    @apply --layout-center;
     background: white;
     transition: background .25s;
 }
 
-app-toolbar[blue] {
-    color: white;
-    background: var(--primary-background-color);
-    --primary-text-color: white;
-    --secondary-text-color: #ffffff69;
-    --primary-color: white;
-    --accent-color: white;
-    --logo-color: white;
-}
-
 app-toolbar>header {
     margin-top: auto;
 }
 
 #Narrow-Slider {
-    position: absolute;
-    left: 0;
-    bottom: 0;
-    top: 0;
-    width: 256px;
```

Comment:
Why did we change the CSS to this? The two changes here are not Mutually Exclusive

Reply:
This is not longer applicable after rebasing.